.jenkins

```diff
@@ -1,16 +1,39 @@
+def getSafeBranchName() {
```

Comment:
Not for this review, but we use these two methods I wrote ages ago in a lot of pipelines, we should try to figure out how to write these methods once and make them a plugin or a shared library.